app/components/export-tool.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    store: Ember.inject.service(),
```

Comment:
This component really should not need to know about the store. Certainly serializing ember-data models is one of our primary use cases, but this could be a much looser coupling.

I'm imagining: 
```handlebars
{{export-tool data=myListOfModels mapping=myMappingFunction}}
```

where myMappingFunction might be like:
```javascript
function(user) { return user.username;}
```

The benefit of this is that the export-tool can be totally agnostic of data source. If you're already on the participants page you probably already have this list available as `model`.